pyshop/views/simple.py

```diff
@@ -287,6 +287,9 @@ def render(self):
 
         package_name = self.request.matchdict['package_name']
         pkg = Package.by_name(self.session, package_name)
+        if pkg is None:
+            pkg = Package.by_name(self.session, package_name.repalce('-', '_'))
```

Comment:
erf sorry i push with the typo /o\

then we have a local egg named xxxx_sptheme
without the replace :
================
pkg is None
pyshop log
----------------
2015-07-21 22:23:37,407 INFO  [pyshop.views.simple][Dummy-2] refresh package xxxx-sptheme
2015-07-21 22:23:38,979 INFO  [pyshop.views.simple][Dummy-2] package xxxx-sptheme has no versions

buildout log
---------------------
Getting distribution for 'xxxx-sptheme==1.2'.
While:
  Installing project.
  Getting distribution for 'xxxx-sptheme==1.2'.
Error: Couldn't find a distribution for 'xxxx-sptheme==1.2'.

with the replace
==============
pyshop log
----------------
2015-07-21 22:28:07,273 INFO  [pyshop.security][Dummy-4] [127.0.0.1] GET http://pypi.intranet/simple/xxxx-sptheme/
2015-07-21 22:28:07,395 INFO  [pyshop.views.base][Dummy-4] dispatch view Show
2015-07-21 22:28:07,407 INFO  [pyshop.security][Dummy-5] [127.0.0.1] GET http://pypi.intranet/repository/10188/xxxx_sptheme-1.2.tar.gz

buildout log
------------------
Got xxxx-sptheme 1.2.